index.html

```diff
@@ -0,0 +1,327 @@
+        </div>
+
+        <div
+          class="
+            container
+            d-flex
+            flex-column flex-md-row flex-wrap
+            justify-content-center justify-content-md-around
+            align-items-center
+          "
+        >
+          <div
+            class="
+              d-flex
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/work.png" alt="Teach" />
+            <h3>Teaching</h3>
+            <p>Word taught in simplicity and truth.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/demonstrator.png" alt="demonstation" />
+            <h3>Healings</h3>
+            <p>Body healed and delivered.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/conversation.png" alt="Believers_meeting" />
+            <h3 class="text-center sp-t">Believers Meeting</h3>
+            <p>Each one has a tongue, interpreatations.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/pray.png" alt="pray" />
+            <h3>Prayer</h3>
+            <p>Heartfelt prayer.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/worship-icon.png" alt="worship" />
+            <h3>Worship</h3>
+            <p>Worship in truth and spirit.</p>
+          </div>
+        </div>
+        <div
+          class="see-height d-flex justify-content-center align-items-center"
+        >
+          <button class="d-lg-none bg-join">
+            Join L2 Ministries Conference 2021
+          </button>
+          <p class="under-p d-none d-lg-block">SEE THE WHOLE PROGRAM</p>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make use of adequate proportionality to space elements in this section, i.e icon size, font weight and padding
_see reference below_
![image](https://user-images.githubusercontent.com/67957533/132743194-fa5ee61a-b0b6-4847-ad0c-38845db18060.png)